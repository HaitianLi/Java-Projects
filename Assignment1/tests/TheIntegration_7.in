add collection books.csv
add collection book2.csv
add book books.csv 111111
add member jackhu
add member jackhu
add member huhuhu
add member mage
member 100000
member 100001
member books 100000
rent 100000 111111
member books 100000
member books 100001
list GENRES
list Authors
Author Haitian li
relinquish all 100000
rent 100001 111111
relinquish all 100001
rent 100002 111111
relinquish all 100002
rent 100003 111111
relinquish all 100003
common 100000 100001 100002 100003
exit
