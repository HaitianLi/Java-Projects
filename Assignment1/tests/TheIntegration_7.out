user: 17 books successfully added.

user: No such collection.

user: Book already exists in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000: jackhu

user: 100001: jackhu

user: Member not currently renting.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Member not currently renting.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: No books by Haitian li.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
